uitools/toolkitcpp/docs/AuthenticationView.md

```diff
@@ -0,0 +1,18 @@
+[![ArcGIS Maps SDK for Qt](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt-0b5394)](https://developers.arcgis.com/qt/) [![ArcGIS Maps SDK for Qt toolkit](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt%20toolkit-ea4d13)](https://github.com/Esri/arcgis-maps-sdk-toolkit-qt) [![Qt Quick UI components](https://img.shields.io/badge/Qt%20Qt%20Quick%20UI%20components-ea4d13)](../../toolkitcpp/)
+
+# AuthenticationView - Qt Quick UI control/QML Type
+
+The [AuthenticationView - Qt Quick UI control/QML Type](https://developers.arcgis.com/qt/toolkit/api-reference/qml-authenticationview.html) handles authentication challenges and automatically launching the appropriate UI for each type of authentication. Declare an AuthenticationView in your QML file. The AuthenticationView will then be connected to all authentication challenges, and will automatically launch the appropriate view for the type of challenge. Supported security formats include:
```

Comment:
Same comment as above could apply here too. Could "Qt Quick UI control/QML Type" appear only either in the heading, or vice-versa, only in the hyperlink?

Reply:
I would like to leave this for now on the individual pages. There is some real confusion between **Qt Quick UI controls** and the **Qt Widgets UI controls**. There are both types of pages in the toolkit doc and they look nearly identical, just hyperlinking to different API Reference docs.